tools/parse-pkgs.sh

```diff
@@ -7,6 +7,9 @@
 EVE="$(cd "$(dirname "$0")" && pwd)/../"
 PATH="$EVE/build-tools/bin:$PATH"
 
+# Version of the Linux kernel that we build for early testing
+NEW_KERNEL_VERSION=5.6.1
```

Comment:
Curious to know why you moved this into here from the Makefile. `parse-pkgs` usually uses logic to determine pkg versions. Hard-coding it in here is going to be hard to track down later.

Reply:
This is basically to make NEW KERNEL a fake package. In fact, doing it convinced me that we should probably make it a real pkg/new-kernel

What do you think @deitch ?